Hangsman/hangman.py

```diff
@@ -0,0 +1,71 @@
+class Hangman:
+
+    def __init__(self, word="Word"):
+        self.word = word.upper()
+        self._guess_count = 6
```

Comment:
we can define 6 as a constant in constant file.
in big projects, we should avoid any magic number